nl.json

```diff
@@ -47,7 +46,7 @@
             },
             "translationNotAvailable" : "Helaas is er geen vertaling beschikbaar voor deze pagina.",
             "translationIncomplete" : "Niet alle tekst op deze pagina zal worden vertaald."
-        },
+        {,
```

Comment:
That was correct before. Your change breaks the whole file, so it can't be used as a translation.